index.html

```diff
@@ -0,0 +1,754 @@
+                                <span>GV</span></i></a>
+                            <div class="border">
+                                <p>Ahana <a href="#" class="faded-out">ZDNet</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="sponsorLast">
+                    <div class="sponsor-image">
+                        <img src="assets/images/sponsor3.jpeg" alt="sponsor image">
+                    </div>
+                    <div class="sponsor-details">
+                        <h3>Zeleros</h3>
+                        <div class="sponsor-assets">
+                            <h4>$7M</h4>
+                            <h5>Series A</h5>
+                        </div>
+                        <div class="sponsor-cards">
+                            <a href="#" class="sponsor-tag"><i class="fas fa-tag"><span> Design, +3</span></i></a>
+                            <a href="#" class="sponsor-tag"><i class="fas fa-briefcase">
+                                <span>Altran, +6</span></i></a>
+                            <div class="border">
+                                <p>Zeleros €7m <a href="#" class="faded-out">UKTN</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <div class="article-section">
+            <section class="popular">
+                <h2>Apps ></h2>
+                <ul class="listOne">
+                    <li>
+                        <div class="sectionPopular2">
+                            <a href="#">
+                                <div class="news__title">
+                                    <h5 class="white no-margin mostPopular">Most popular</h5>
+                                    <h3 class="white no-margin">Pablo Escobar’s brother is suing Apple for $2.6B</h3>
+                                    <div class="news__author no-margin padding-top">
+                                        Callom Booth. 5 hours ago
+                                    </div>
+                                </div>
+                            </a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="latest">
+                            <div class="laTitles">
+                                <h5 class="no-margin">LATEST</h5>
+                                <h4 class="no-margin">Spotify’s removed its 10K library limit — but it won’t replace my
+                                    MP3
+                                    collection </h4>
+                            </div>
+                            <img src="assets/images/icon1.png" alt="icon 1">
+                        </div>
+                    </li>
+                </ul>
+                <ul class="listTwo">
```

Comment:
This and all other -> `.listTwo`, texts inside them are too small to read. Increase their font size.